static/js/modules/columns.js

```diff
@@ -15,6 +15,26 @@ var sizeInfo = {
   2000: {limits: [2000, null], label: 'Over $2000'},
 };
 
+var supportOpposeMap = {
+  S: 'Support',
+  O: 'Oppose',
+};
+
+var supportOpposeColumn = {
+  data: 'support_oppose_indicator',
+  render: function(data, type, row, meta) {
```

Comment:
'meta' is defined but never used.<br>'row' is defined but never used.<br>'type' is defined but never used.